plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestTimestampMicros.java

```diff
@@ -84,11 +70,15 @@ public void testTimestampMicrosAsTimestampWithTimeZone(HiveTimestampPrecision ti
         File parquetFile = new File(Resources.getResource("issue-5483.parquet").toURI());
         Type columnType = createTimestampWithTimeZoneType(timestampPrecision.getPrecision());
 
-        try (ConnectorPageSource pageSource = createPageSource(session, parquetFile, "created", HIVE_TIMESTAMP, columnType)) {
-            MaterializedResult result = materializeSourceDataStream(session, pageSource, List.of(columnType)).toTestTypes();
-            assertThat(result.getMaterializedRows())
-                    .containsOnly(new MaterializedRow(List.of(expected.atZone(ZoneId.of("UTC")))));
-        }
+//      TODO: revise this test when `timestamp with time zone` will be supported by Hive connector, follow by: https://github.com/trinodb/trino/issues/1240
```

Comment:
It seems here that the test has degraded now because it can't perform anymore the initial assertion.
Probably doing the refactoring is not necessarily a good idea.

Reply:
any suggestions?